arangod/Metrics/MetricsFeature.cpp

```diff
@@ -365,4 +388,12 @@ void MetricsFeature::batchRemove(std::string_view name,
   }
 }
 
+void MetricsFeature::prepare() {
+  _queryRegistryFeature = std::move(_lazyQueryRegistryFeatureRef).get();
+  _statisticsFeature = std::move(_lazyStatisticsFeatureRef).get();
+  _engineSelectorFeature = std::move(_lazyEngineSelectorFeatureRef).get();
+  _clusterMetricsFeature = std::move(_lazyClusterMetricsFeatureRef).get();
+  _clusterFeature = std::move(_lazyClusterFeatureRef).get();
+}
+
```

Comment:
Should we assert somehow that `prepare()` and therefore `get()` methods are really called only once?

Reply:
It is asserted that `get()` is called only once: https://github.com/arangodb/arangodb/pull/20945/files?show-viewed-files=true&file-filters%5B%5D=#diff-ba7c32ee3716b0719387545111a837bd0a213f052957a14db2d32b9ee24be3ccR103